CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### [2.9.0](https://github.com/Kozea/formol/compare/v2.8.3...v2.9.0)
+
+- Replace native date input by react-datepicker
+
```

Comment:
I think this should be in master branch after merging the PR, as well as the version publishing commit.